DPG/Tasks/qaMatchEff.cxx

```diff
@@ -420,8 +478,32 @@ struct qaMatchEff {
           //
       }   // end pions only
       //
+      // no primary/sec-d pions
+      if (!((tpPDGCode == 211) && (mcpart.isPhysicalPrimary()))) {
+        // gets the pdg code and finds its index in our vector
+        itr_pdg = std::find(pdgChoice.begin(), pdgChoice.end(), tpPDGCode);
+        if (itr_pdg != pdgChoice.cend())
+          // index from zero, so increase by 1 to put in the right bin (and 0.5 not needed but just not to sit in the edge)
+          pdg_fill = (float)std::distance(pdgChoice.begin(), itr_pdg) + 1.5;
+        else
+          pdg_fill = -10.0;
```

Comment:
This `-10.0` goes in the underflow, so if I understand properly:
 - with 0 you keep flag fake tracks
 - with -10 those with a MC particle that is not in the bunch of those you decided to check.
Is it correct?